lua/lvim/core/autocmds.lua

```diff
@@ -34,6 +34,59 @@ function M.load_defaults()
         end,
       },
     },
+    {
+      "BufReadPre",
+      {
+        group = "_general_settings",
+        pattern = "*",
+        desc = "Performance improvements for big files",
+        callback = function(args)
```

Comment:
we should move this into its own function just for the sake of readability, maybe even its own file.